src/commands/math/basicSymbols.js

```diff
@@ -526,3 +528,26 @@ LatexCmds['÷'] = LatexCmds.div = LatexCmds.divide = LatexCmds.divides =
   bind(BinaryOperator,'\\div ','&divide;', '[/]');
 
 CharCmds['~'] = LatexCmds.sim = bind(BinaryOperator, '\\sim ', '~', '~');
+
+for (var cmd in LatexCmds) {
+  if (LatexCmds.hasOwnProperty(cmd)) {
+      constructor = LatexCmds[cmd];
+      var block = constructor();
+      // find templates with a unicode character code
+      // we can programatically generate the character in a string
+      var findUnicodePattern = /&#(.*);/;
```

Comment:
technically this is called a "numeric character reference" and the other one a "named character (entity) reference" but I'm not sure those are actually more legible lmao

https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references